lists/broken

```diff
@@ -82,6 +85,7 @@ pth-toolkit # abandoned, fails in run due to upstream compiling by old libs, bet
 pyersinia
 pyqt3 # broken, discontinued, not needed anymore, see #3660
 pytbull # discontinued, better delete
+python-boto # #4141
```

Comment:
there is no issue with that number

Reply:
Look the last one in the image (that is #4141). Probably the related issue has been wrongly canceled.

<img width="949" height="373" alt="image" src="https://github.com/user-attachments/assets/345f6834-60c3-43c3-87fa-0e787e050464" />

Reply:
weird, must have been remove